tap-version-14-specification.md

```diff
@@ -0,0 +1,1167 @@
+```
+
+## Procrastination Considered `ok`
+
+The following example reports that four tests are run and the last two
+tests failed. However, because the failing tests are marked as things to do
+later, they are considered successes. Thus, a harness should report this
+entire listing as a success.
+
+```tap
+TAP version 14
+1..4
+ok 1 - Creating test program
+ok 2 - Test program runs, no error
+not ok 3 - infinite loop # TODO halting problem unsolved
+not ok 4 - infinite loop 2 # TODO halting problem unsolved
+```
+
+## Creative liberties
+
+This listing shows an alternate output where the test numbers aren't
+provided. The test also reports the state of a ficticious board game as a
+YAML block. Finally, the test count is reported at the end.
+
+```tap
+TAP version 14
+ok - created Board
+ok
+ok
+ok
+ok
+ok
+ok
+ok
+  ---
+  message: "Board layout"
+  severity: comment
+  dump:
+     board:
+       - '      16G         05C        '
+       - '      G N C       C C G      '
+       - '        G           C  +     '
+       - '10C   01G         03C        '
+       - 'R N G G A G       C C C      '
+       - '  R     G           C  +     '
+       - '      01G   17C   00C        '
+       - '      G A G G N R R N R      '
+       - '        G     R     G        '
+  ...
+ok - board has 7 tiles + starter tile
+1..9
+```
+
+## Bugs
+
+Feature requests and bug reports should be [raised on
+GitHub.](https://github.com/TestAnything/Specification/issues/new)
+
+## Authors
+
+The TAP 14 Specification is authored by [Isaac Z.
+Schlueter](https://github.com/isaacs), as a result of much discussion on
+the [TestAnything
+Specification](https://github.com/TestAnything/Specification) project, with
+considerable input, encouragement, feedback, and suggestions from:
+
+* [Matt Layman](https://github.com/mblayman)
+* [Leon Timmermans](https://github.com/Leont)
+* [Bruno P. Kinoshita](https://github.com/kinow)
+* [Chad Granum](https://github.com/exodist)
+
+## Acknowledgements
+
+Special thanks to [Jonathan Kingston](https://github.com/jonathanKingston)
+for his efforts to keep TAP flourishing and bring implementors together.
+
+The TAP 13 Specification was written by Andy Armstrong with help and
+contributions from Pete Krawczyk, Paul Johnson, Ian Langworth and Nik
+Clayton, based on the original TAP documentation by Andy Lester, based on
+the original `Test::Harness` documentation by Michael Schwern.
+
+The basis for the TAP format was created by Larry Wall in the original test
+script for Perl 1. Tim Bunce and Andreas Koenig developed it further with
+their modifications to `Test::Harness`.
+
+## Copyright
+
+Copyright 2015-2022 by Isaac Z. Schlueter <i@izs.me> and Contributors.
+
+Copyright 2003-2007 by Michael G Schwern <schwern@pobox.com>, Andy Lester
+<andy@petdance.com>, Andy Armstrong <andy@hexten.net>.
+
+This specification is released under the [Artistic License
+2.0](https://opensource.org/licenses/Artistic-2.0)
```

Comment:
TIL about Artistic-2.0. I [think](https://www.acc.com/sites/default/files/2021-01/Crowell%20%26%20Moring%20-%20ACC%20SFBA%20Free%20and%20Open%20Source%20License%20Summary%20-%20Features%20that%20Define%20Strategy.pdf) it's similar to creative commons, but people using the content must include license, disclose source, and tell us about any changes made. :+1:

Reply:
Yes, essentially it's permissive but you can't change it (meaningfully) and claim it's the same thing.  Have to either send the changes upstream, or call it by a new name.